include/blmc_robots/common_header.hpp

```diff
@@ -48,6 +48,11 @@ typedef Eigen::Matrix<double, 2, 1> Vector2d;
  */
 typedef Eigen::Matrix<double, 8, 1> Vector8d;
 
+/**
+ * @brief Vector8d shortcut for the eigen vector of size 12.
+ */
+typedef Eigen::Matrix<double, 12, 1> Vector12d;
```

Comment:
I know we did not do it for the other robot but should we call this solo::vector? With a namespace for each robot.